mu-plugins/blocks/multisite-latest-posts/multisite-latest-posts.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * Plugin Name: Multisite Latest Posts
+ * Description: A block for use across the whole wp.org network.
+ *
+ * @package wporg
+ */
+
+namespace WordPressdotorg\MU_Plugins\wporg;
+
+/**
+ * Helper function to find item in array by id.
+ *
+ * @param WP_Term[] $arr List of categories
+ * @param number    $id Category ID
+ * @return WP_Term|string
+ */
+function get_by_id( $arr, $id ) {
```

Comment:
I feel there's a better PHP way of doing this :).

Reply:
https://developer.wordpress.org/reference/functions/wp_list_filter/ maybe

Reply:
That function probably does more than necessary. I think I'll just stick to this helper.

Reply:
FWIW I agree. although `return wp_list_filter( $arr, [ 'id' => $id ] )[0] ?? '';` doesn't have _that much_ overhead, but at the same time, doesn't make it any more readable.

Reply:
Yeah, just needing to create a [wp_list_util](https://developer.wordpress.org/reference/classes/wp_list_util/) object just to filter an array seems a bit much.

Reply:
Might as well just use `array_filter`...?